xbmc/music/Album.cpp

```diff
@@ -200,9 +199,12 @@ const std::string CAlbum::GetAlbumArtistString() const
   //but is takes precidence as a string because artistcredits is not always filled during processing
   if (!strArtistDesc.empty())
     return strArtistDesc;
+  std::vector<std::string> artistvector;
```

Comment:
std::vector<std::string> artistvector(artistCredits.size()); to set the size directly as we know it